_posts/work/2018-01-31-credit-models-with-dask.md

```diff
@@ -140,15 +164,17 @@ avg_default.visualize()
 
 And that is how dask can be used to construct a complex system of equations with reusable intermediary calculations.
 
-## How we used dask
 
-For our credit modeling problem, we used dask to make a custom data structure to represent the individual equations. Using the default example above, this looked something like:
+How we used dask in practice
```

Comment:
rst header?

Reply:
This is also valid markdown.  I tend to prefer these to `##` just because they break up the source file with a bit more visual weight.  Regardless, I've switched to `##` for consistency.